### PR TITLE
Add dfeshiny package

### PR DESCRIPTION
**Pull request by rmbielby**
## Pull request overview

The dfeshiny package only has functions to run tidy_code() and support_panel() so far, but I've wired these up to keep momentum. This version of the template allows the user to run the dfeshiny version of tidy code (with the local code having been removed) and renders the support panel using the dfeshiny function.

Aiming to add the updated commit hooks in as well from the KS4 dashboard, which would complement this as that uses tidy_code().

## Pull request checklist

Please check if your PR fulfils the following:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Tests have been run locally and are passing (`run_tests_locally()`)
- [ ] Code is styled according to tidyverse styling (checked locally with `tidy_code()`)

### CODE REVIEW

File: .Rprofile [hunk 1]
```diff
@@ -12,29 +12,5 @@ cat("Sourcing .Rprofile.", fill = TRUE)
...
 source("renv/activate.R")
 
-# Tidy code function
-tidy_code <- function() {
-  source("global.r")
-  tidy_code_function()
-}
-
-# Function to run tests
-run_tests_locally <- function() {
```
Comment (cjrace) [inline .Rprofile:22]: Why did you remove this function?

**Review by cjrace — CHANGES_REQUESTED** [review]

File: .Rprofile (hunk 1 above)
Comment (rmbielby) [reply .Rprofile:22]: Good question... took it out when playing around with the dfeshiny package and should have restored it. Have added it back in now.

File: .Rprofile [hunk 2]
```diff
@@ -12,29 +12,5 @@ cat("Sourcing .Rprofile.", fill = TRUE)
 
 source("renv/activate.R")
 
-# Tidy code function
```
[the remark this answers is not in the document]
Comment (rmbielby) [reply .Rprofile:15]: Added.

[the remark this answers is not in the document]
Comment (rmbielby) [reply .Rprofile:15]: Actually that broke the scripts. I've put a prompt to the user to run renv::restore() if dfeshiny isn't available. So now .Rprofile will load it if it is installed and will give a warning if it isn't. And dfeshiny will then get loaded once the user has restored the renv snapshot and then run the app (as dfeshiny is loaded in global.R anyway).

**Review by cjrace — CHANGES_REQUESTED** [review]
One last small thing before I approve - we seem to get a random line of guidance appear when running the tests? 

image

**Review by cjrace — APPROVED** [review]

**Pull request merged**